Sources/Core/Shared/OperationQueue.swift

```diff
@@ -197,7 +197,7 @@ public extension OperationQueue {
 
      - returns: The main queue
      */
-    public override class func main() -> OperationQueue {
+    public class func main() -> OperationQueue {
```

Comment:
This might be incorrect? The return type should be `Operations.OperationQueue`. Not sure if that will need an override though.